### PR TITLE
[TW-808] flows - send info

### PR DESCRIPTION
**Pull request by petewitcher**
This is a new tutorial for Postman Flows that shows how to take response data from one GET request and include it in a subsequent GET request.

### CODE REVIEW

**Review by DanielKimmelmann1 — APPROVED** [review]
LGTM!

File: src/pages/docs/postman-flows/flows-intro/using-response-data-in-a-request.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+The Random User API returns a randomly chosen set of user data for each GET request, unless you include a seed as a query parameter. This tutorial takes the seed from a GET response and includes it as a query parameter in another GET request so that both GET requests return the same user’s data.
+
+## Contents
+
+* [Create a collection and a request](#create-a-collection-and-a-request)
+* [Create a flow](#create-a-flow)
+
+## Create a collection and a request
+
+1. Create a collection named `Random User Collection` and add a GET request named `First Request`.
```
Comment (jkonrath-postman) [inline src/pages/docs/postman-flows/flows-intro/using-response-data-in-a-request.md:17]: Just a thought to note for later - how about we make these collections and publish them somewhere? We can still show the steps here of what exactly to do, but people could fork the examples if they want to. Let's talk about it offline.

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/postman-flows/flows-intro/using-response-data-in-a-request.md (hunk 1 above)
Comment (petewitcher) [reply src/pages/docs/postman-flows/flows-intro/using-response-data-in-a-request.md:17]: Great idea!

**Pull request merged**